Meta/Lagom/Tools/CodeGenerators/LibUnicode/GenerateUnicodeData.cpp

```diff
@@ -451,6 +486,25 @@ static void add_canonical_code_point_name(CodePointRange range, StringView name,
     unicode_data.code_point_display_names.append({ range, name });
 }
 
+static void parse_block_display_names(Core::File& file, UnicodeData& unicode_data)
```

Comment:
FYI there was a change on `master` where we now use the `Core::Stream` API to parse these files, so this will need to be rebased to do the same (sorry, bad timing).